src/components/Header.jsx

```diff
@@ -0,0 +1,31 @@
+import style from "./Header.module.scss";
+import iconMoonNight from "../assets/images/half-moon-shape-svgrepo-com.svg";
```

Comment:
Use it as a component. With vite you need to find out how exactly you can achieve that.

Reply:
done